cloudbuild-prod.yaml

```diff
@@ -34,6 +34,12 @@ steps:
         "internal-and-cloud-load-balancing",
         "--update-env-vars",
         "SDS_APPLICATION_VERSION=${TAG_NAME}",
```

Comment:
It's probably fine to set min to 2 since you get charged for use rather than them existing